content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -102,7 +102,7 @@ kube-apiserver -h | grep enable-admission-plugins
 In Kubernetes {{< skew currentVersion >}}, the default ones are:
 
 ```shell
-CertificateApproval, CertificateSigning, CertificateSubjectRestriction, DefaultIngressClass, DefaultStorageClass, DefaultTolerationSeconds, LimitRanger, MutatingAdmissionWebhook, NamespaceLifecycle, PersistentVolumeClaimResize, PodSecurity, Priority, ResourceQuota, RuntimeClass, ServiceAccount, StorageObjectInUseProtection, TaintNodesByCondition, ValidatingAdmissionWebhook
+CertificateApproval, CertificateSigning, CertificateSubjectRestriction, DefaultIngressClass, DefaultStorageClass, DefaultTolerationSeconds, LimitRanger, MutatingAdmissionWebhook, NamespaceLifecycle, PersistentVolumeClaimResize, PodSecurity, Priority, ResourceQuota, RuntimeClass, ServiceAccount, StorageObjectInUseProtection, TaintNodesByCondition, ValidatingAdmissionPolicy, ValidatingAdmissionWebhook
```

Comment:
This is a list of default enabled plugins right?

Reply:
That's a good point. Catch is that we do want to document the order somewhere if possible..   If this is the only list that states the exact order, maybe we can put a comment in parens that this feature is gated?

Reply:
Or maybe we just wait till the feature is on by default to document order?

Reply:
There's a section below `What does each admission controller do?` that lists all admission controls (including disabled by default), maybe just add it there?

Reply:
Sounds great to me. Thanks for hunting this down.

Reply:
The ValidatingAdmissionPolicy is actually enabled in admission chain, but the underline validation is feature gated. In this case it might make sense to add it here as well. I will add more info in `What does each admission controller do?` section.